packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/ReviewWorkflows.js

```diff
@@ -1,35 +1,35 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useIntl } from 'react-intl';
+import { useSelector, useDispatch } from 'react-redux';
 import { SettingsPageTitle } from '@strapi/helper-plugin';
-import { Button, ContentLayout, HeaderLayout, Layout, Main } from '@strapi/design-system';
+import { Button, ContentLayout, HeaderLayout, Layout, Loader, Main } from '@strapi/design-system';
 import { Check } from '@strapi/icons';
 
 import { Stages } from './components/Stages';
+import { reducer } from './reducer';
+import { REDUX_NAMESPACE } from './constants';
+import { useInjectReducer } from '../../../../../../admin/src/hooks/useInjectReducer';
+import { useReviewWorkflows } from './hooks/useReviewWorkflows';
+import { setWorkflow } from './actions';
 
-const STAGES = [
-  {
-    uid: 'id-1',
-    name: 'To do',
-  },
-
-  {
-    uid: 'id-2',
-    name: 'Ready to review',
-  },
+export function ReviewWorkflowsPage() {
+  const { formatMessage } = useIntl();
+  const { workflows: workflowsData } = useReviewWorkflows();
+  const workflow = useSelector((state) => state?.[REDUX_NAMESPACE]);
```

Comment:
It is actually helpful to know it can be undefined, because it is a way to understand `useInjectReducer` didn't run yet (because initial state is not applied). What benefit do you see in returning an object?

Reply:
What do you think about this instead? https://github.com/strapi/strapi/pull/15635/commits/d9af030383a354d2679acf19ef12e56a6826684e

Reply:
> It is actually helpful to know it can be undefined, because it is a way to understand useInjectReducer didn't run yet (because initial state is not applied). What benefit do you see in returning an object - just the fact I could get workflows directly by destructuring?

Pretty much, i just didn't like the `workflows.workflows` 😂  total preference. So yes, your commit makes me happy 👍🏼